scripts/materialize_b100_sql_parity_fixtures.py

```
@@ -0,0 +1,208 @@
+
+def econ2() -> None:
+    cols = (
+        "col,point_role\n"
+        "oa_t,outside-air-temp\n"
+        "oa_damper_pct,outside-air-damper\n"
+        "fan_status,fan-status\n"
+        "fan_cmd,fan-cmd\n"
+    )
+    header = "timestamp_utc,oa_t,oa_damper_pct,fan_status,fan_cmd"
+    times = ts_rows(24)
+    # 20% OA (0–100 scale) is not > 0.42 after /100.
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)